acceptance/openstack/identity/v3/identity.go

```diff
@@ -166,3 +182,71 @@ func DeleteDomain(t *testing.T, client *gophercloud.ServiceClient, domainID stri
 
 	t.Logf("Deleted domain: %s", domainID)
 }
+
+// UnassignRoleFromUserOnProject will revoke a role of a user on a project. A fatal error will
+// occur if the revoke was unsuccessful. This works best when used as a deferred function
+func UnassignRoleFromUserOnProject(t *testing.T, client *gophercloud.ServiceClient, role *roles.Role, user *users.User, project *projects.Project) {
+	t.Logf("Attempting to remove role %s from user %s on project %s", role.Name, user.Name, project.Name)
+
+	err := roles.UnassignFromUserOnProject(client, role.ID, user.ID, project.ID).ExtractErr()
+	if err != nil {
+		t.Fatalf("Unable to remove role")
+	}
+
+	t.Logf("Removed role %s from user %s on project %s", role.Name, user.Name, project.Name)
+}
+
+// FindRole finds all roles that the current authenticated client has access
+// to and returns the first one found. An error will be returned if the lookup
```

Comment:
Should the function name reflect that 'Any' (or 'First' of 'One') role is being returned?

Reply:
We're quite lenient with functions used for acceptance testing. As long as it doesn't blow the cloud up, I'm OK with it. :)

Reply:
@jtopjian Just curious, is this function in a section that calls out "test purpose only"? It is not obvious that this function is meant for acceptance testing only. Perhaps a comment would do for now.

Reply:
Nevermind, I didn't see the 'acceptance' at the very top of the folder structure.